Extension/src/LanguageServer/client.ts

```diff
@@ -134,7 +134,65 @@ function collectSettings(filter: (key: string, val: string, settings: vscode.Wor
         }
         if (filter(key, val, settings)) {
             previousCppSettings[key] = val;
-            result[key] = (key === "clang_format_path") ? "..." : String(previousCppSettings[key]);
+            switch (key.toLowerCase()) {
+                case "clang_format_path": {
+                    if (val) {
+                        switch (val.toLowerCase()) {
```

Comment:
`val` is `any`, so I don't think you can safely call this without converting to string first.